nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/recipe/HelmHostedRecipe.groovy

```diff
@@ -0,0 +1,260 @@
+package org.sonatype.repository.helm.internal.content.recipe
+
+import javax.annotation.Nonnull
+import javax.inject.Inject
+import javax.inject.Named
+import javax.inject.Provider
+import javax.inject.Singleton
+
+import org.sonatype.nexus.repository.Format
+import org.sonatype.nexus.repository.RecipeSupport
+import org.sonatype.nexus.repository.Repository
+import org.sonatype.nexus.repository.Type
+import org.sonatype.nexus.repository.http.HttpHandlers
+import org.sonatype.nexus.repository.http.PartialFetchHandler
+import org.sonatype.nexus.repository.httpclient.HttpClientFacet
+import org.sonatype.nexus.repository.security.SecurityHandler
+import org.sonatype.nexus.repository.types.HostedType
+import org.sonatype.nexus.repository.view.ConfigurableViewFacet
+import org.sonatype.nexus.repository.view.Context
+import org.sonatype.nexus.repository.view.Matcher
+import org.sonatype.nexus.repository.view.Route
+import org.sonatype.nexus.repository.view.Router
+import org.sonatype.nexus.repository.view.ViewFacet
+import org.sonatype.nexus.repository.view.handlers.ConditionalRequestHandler
+import org.sonatype.nexus.repository.view.handlers.ContentHeadersHandler
+import org.sonatype.nexus.repository.view.handlers.ExceptionHandler
+import org.sonatype.nexus.repository.view.handlers.FormatHighAvailabilitySupportHandler
+import org.sonatype.nexus.repository.view.handlers.HandlerContributor
+import org.sonatype.nexus.repository.view.handlers.HighAvailabilitySupportChecker
+import org.sonatype.nexus.repository.view.handlers.TimingHandler
+import org.sonatype.nexus.repository.view.matchers.ActionMatcher
+import org.sonatype.nexus.repository.view.matchers.LiteralMatcher
+import org.sonatype.nexus.repository.view.matchers.logic.LogicMatchers
+import org.sonatype.nexus.repository.view.matchers.token.TokenMatcher
+import org.sonatype.repository.helm.internal.AssetKind
+import org.sonatype.repository.helm.internal.HelmFormat
+import org.sonatype.repository.helm.internal.content.HelmContentFacet
+import org.sonatype.repository.helm.internal.content.createindex.CreateIndexFacetImpl
+import org.sonatype.repository.helm.internal.hosted.HostedHandlers
+import org.sonatype.repository.helm.internal.security.HelmSecurityFacet
+
+import static org.sonatype.nexus.repository.http.HttpMethods.DELETE
+import static org.sonatype.nexus.repository.http.HttpMethods.GET
+import static org.sonatype.nexus.repository.http.HttpMethods.HEAD
+import static org.sonatype.nexus.repository.http.HttpMethods.PUT
+import static org.sonatype.repository.helm.internal.AssetKind.HELM_INDEX
+import static org.sonatype.repository.helm.internal.AssetKind.HELM_PACKAGE
+import static org.sonatype.repository.helm.internal.AssetKind.HELM_PROVENANCE
+
+/**
+ * Helm Hosted Recipe
+ *
+ * @since 1.0.11
```

Comment:
Maybe this is to be fixed up later but https://github.com/sonatype-nexus-community/nexus-repository-helm/pull/137/files#diff-7c13d52dac570e30f23434c2d2392d95R24 says 1.0.10, so it seems inconsistent?

Did one come before 3.24 and the other after, maybe?

Reply:
Now version is 1.0.11-SNAPSHOT so in best case it will be released in 1.0.11. Aligned 5c18845436092fd10ac264980135f8b2edb04ce4